packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -91,7 +92,14 @@ const ConfirmationModal = ({ is_visible, toggleModal, onSubmit }) => (
 
 const ConfirmationPage = ({ toggleModal, onSubmit }) => (
     <div className='account__confirmation-page'>
-        <span className='account__confirmation-page-title'>{localize('Notice')}</span>
+        <Text
+            size='xs'
+            weight='bold'
+            styles={{ color: 'var(--brand-red-coral)' }}
+            className='account__confirmation-page-title'
```

Comment:
Is there any reason to remove `color` from the CSS class and add it as `style`?

Reply:
Please check here:
https://github.com/binary-com/deriv-app/pull/2141#discussion_r528613723